pulsar-client/src/test/java/org/apache/pulsar/client/impl/auth/oauth2/AuthenticationOAuth2Test.java

```diff
@@ -86,6 +86,7 @@ public void testConfigure() throws Exception {
         params.put("privateKey", "data:base64,e30=");
         params.put("issuerUrl", "http://localhost");
         params.put("audience", "http://localhost");
+        params.put("scope", "test-scope");
```

Comment:
can you please add more testing about this value ?
I see this only added here and there is not follow up verification

Reply:
@addisonj  I have added some tests, please take a look at this, thanks